src/blocks/components/components-panel/components-panel-editor.scss

```
@@ -0,0 +1,8 @@
+.components-panel {
+  .components-notice {
+    &.is-dismissible {
+      padding-right: 0;
+      margin: 10px 0;
+    }
+  }
+}
```

Should the notice be its own element? Like in a standalone file?

A panel is a container. Or did you meant

```suggestion
.components-panel {
  &__notice {
    &.is-dismissible {
      padding-right: 0;
      margin: 10px 0;
    }
  }
}
```

---

Notice is default Gutenberg component (https://developer.wordpress.org/block-editor/components/notice/) and it looks ugly when used in sidebar.

But it does make sense to add this style as the style of that component (`components/notice/notice-style-editor.scss`), will do that! 👍

---

Oh, didn't know that it's a core element. They have ugly css names 😂

---

Yeah, fake BEM xD